yarn.lock

```diff
@@ -28812,6 +28828,32 @@ fsevents@~2.1.2:
   languageName: node
   linkType: hard
 
+"playwright-core@npm:=1.16.3":
+  version: 1.16.3
+  resolution: "playwright-core@npm:1.16.3"
+  dependencies:
+    commander: ^8.2.0
+    debug: ^4.1.1
+    extract-zip: ^2.0.1
+    https-proxy-agent: ^5.0.0
+    jpeg-js: ^0.4.2
+    mime: ^2.4.6
+    pngjs: ^5.0.0
+    progress: ^2.0.3
+    proper-lockfile: ^4.1.1
+    proxy-from-env: ^1.1.0
+    rimraf: ^3.0.2
+    socks-proxy-agent: ^6.1.0
+    stack-utils: ^2.0.3
+    ws: ^7.4.6
+    yauzl: ^2.10.0
+    yazl: ^2.5.1
+  bin:
+    playwright: cli.js
+  checksum: 9ae0cc2714d24fc464ab29cd9c9f4ed2b6d6cf0d88ed1034d40763501f4a9c5094bdff2008c1eb23123671eb61590b0c71af1971714d3b0b7541e42d3ba0d519
+  languageName: node
+  linkType: hard
+
 "playwright@npm:1.14.0":
```

Comment:
Some package (desktop tests?) is still requesting Playwrgith 1.14.0. It would be awesome if we can update all usages of Playwright to keep the dependency tree simpler.

Reply:
I was going to ping you to ask if the Desktop tree should also be updated to 1.16. Is there a way to update the dependency in one location instead of three?

Reply:
No there isn't, as far as I know. What I usually do is a search&replace in all `package.json` files, and then run `yarn` to let it update `yarn.lock`.

Reply:
TIL! `yarn up playwright@1.16` worked perfectly!